src/main/java/org/hawkular/client/android/backend/BackendClient.java

```diff
@@ -136,14 +136,15 @@ public void configureCommunication(@NonNull String host,
 
     private void configurePipes(URL backendUrl, Persona persona) {
         URL pipeUrl = Urls.getUrl(backendUrl, BackendPipes.Paths.ROOT);
-
+        URL alertResolveUrl = Urls.getUrl(pipeUrl, BackendPipes.Paths.ALERT_RESOLVE);
+        URL alertAckUrl = Urls.getUrl(pipeUrl, BackendPipes.Paths.ALERT_ACKNOWLEDGE);
         List<PipeModule> pipeModules = Arrays.asList(
             getAuthorizationModule(),
             getPersonnelModule(persona));
 
         configurePipe(BackendPipes.Names.ALERTS, pipeUrl, pipeModules, Alert.class);
-        configurePipe(BackendPipes.Names.ALERT_ACKNOWLEDGE, pipeUrl, pipeModules, String.class);
-        configurePipe(BackendPipes.Names.ALERT_RESOLVE, pipeUrl, pipeModules, String.class);
+        configurePipe(BackendPipes.Names.ALERT_ACKNOWLEDGE, alertAckUrl, pipeModules, String.class);
```

Comment:
Didn't alter url for persona in fear of hearing it be useless change. cause will have no extra effect :smile: